cluster/gce/configure-vm.sh

```diff
@@ -242,13 +246,20 @@ EOF
 
 # This should only happen on cluster initialization
 function create-salt-auth() {
```

Comment:
I find it super helpful when functions list "assumed vars" and "set vars." I'm definitely not asking you to re-audit the whole file, but could you maybe add a note that `create-salt-auth` should be called first to set `$KUBELET_TOKEN`?